sphinx/util/docfields.py

```diff
@@ -57,7 +59,7 @@ class Field:
     def __init__(
         self,
         name: str,
-        names: tuple[str, ...] = (),
+        names: Sequence[str] = (),
```

Comment:
I'd prefer keeping `tuple[str, ...]`, because a string is considered a sequence of string. It's better to explicitly pass a `tuple` at the call site (where are there?)